dotnet/samples/KernelSyntaxExamples/Example60_HuggingFaceEmbedding.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel;
+
+public static class Example60_HuggingFaceEmbedding
```

Comment:
We already have `Example20_HuggingFace` file with HuggingFace examples, so we can place all new examples related to HuggingFace there.